### DISCUSSION TITLE
Unreal Engine Mouse/Keyboard Controlled Flight

### DISCUSSION THREAD

**Discussion by Zaryob** [JSBSim-Team/jsbsim#822] [Q&A] [answered]
I'm trying to recognize how can I fly in Reference App. Do I need to modify anything or there are already a way to fly?

**Comment by gallonmate** [comment]
https://github.com/JSBSim-Team/jsbsim/tree/master/UnrealEngine#flight-commands

**Comment by Zaryob** [comment]
Yeah. I see it. Basically I dont understand what will ı do after run engines

**Comment by gallonmate** [comment, marked as answer]
**Throttle - Full**, press key 4. This will increase engine to full power for takeoff.

**Reply by Zaryob** [reply]
Thanks I have no idea about aviation.
